templates/mks/party_detail.html

```diff
@@ -81,21 +82,45 @@
                     </h2>
                 </header>
                 <ul>
-                {% for agenda in agendas %}
-                    <li>
-                        <p class="item-title"><a href="{% url 'agenda-detail' agenda.id %}">{{ agenda }}</a></p>
-                        <div class="row">
-                            <span class="span1 numeric text-{% if agenda.score < 0 %}error{%else%}success{% endif %}">
-                                {{agenda.score|floatformat:"0"}}%
+                    {% for agenda in agendas %}
+                        <li>
+                            <p class="item-title"><a href="{% url 'agenda-detail' agenda.id %}">{{ agenda }}</a></p>
+                            <div class="row">
+                            <span class="span1 numeric text-{% if agenda.score < 0 %}error{% else %}success{% endif %}">
+                                {{ agenda.score|floatformat:"0" }}%
                             </span>
-                            <span class="span4">{% bar agenda.score -100 100%}</span>
-                        </div>
-                    </li>
-                {% endfor %}
+                                <span class="span4">{% bar agenda.score -100 100 %}</span>
+                            </div>
+                        </li>
+                    {% endfor %}
+                </ul>
+            </section>
+
+            <section class="card card-list card-list-latest" id="kikar-statuses-section" style="display: none"
+                     data-type="party" data-id="{{ object.id }}" data-test-name="{{ object.name }}">
+                {% include 'kikar/_facebook_status_section_update_elem.html' %}
+                <header>
+                    <h2>
+                        <i class="fa fa-facebook-square"></i>
+                        {% trans "Facebook Statuses" %}
+                    </h2>
+                    <!--a class="btn btn-mini btn-question" href="#">?</a-->
+                </header>
+                <ul id="kikar-facebook-updates-ul"
+                    data-filter='feed__persona__object_id__in={{ object.current_members| objects_to_ids | join:"," }}'>
```

Comment:
@yotammanor is there way to move this kind of query logic to the view? as a method on the "object" or some other solution and not keep in the template? hard to reason about this kind of code in templates and hard to cache (correctly)  or scale.